bnd.bnd

```diff
@@ -0,0 +1,17 @@
+Bundle-Category: sling
```

Comment:
Please don't duplicate meta information already set in parent

Reply:
Removed duplicated information